### PR TITLE
changed ml to ai, bumped year to 2020

### PR DESCRIPTION
**Pull request by kamil-kaczmarek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [![Downloads](https://pepy.tech/badge/neptune-client/week)](https://pepy.tech/project/neptune-client/week)
 
 [![PyPI version](https://badge.fury.io/py/neptune-client.svg)](https://badge.fury.io/py/neptune-client)
-[![Build Status](https://travis-ci.org/neptune-ml/neptune-client.svg?branch=master)](https://travis-ci.org/neptune-ml/neptune-client)
+[![Build Status](https://travis-ci.org/neptune-ai/neptune-client.svg?branch=master)](https://travis-ci.org/neptune-ai/neptune-client)
```
Comment (hubertjaworskineptune) [inline README.md]: revert until github organization changes

File: neptune/__init__.py [hunk 2]
```diff
@@ -37,7 +37,7 @@
 
 
 def init(project_qualified_name=None, api_token=None, proxies=None, backend=None):
-    """Initialize `Neptune client library <https://github.com/neptune-ml/neptune-client>`_ to work with
+    """Initialize `Neptune client library <https://github.com/neptune-ai/neptune-client>`_ to work with
```
Comment (hubertjaworskineptune) [inline neptune/__init__.py]: revert until github organization changes

File: neptune/__init__.py [hunk 3]
```diff
@@ -146,7 +146,7 @@ def init(project_qualified_name=None, api_token=None, proxies=None, backend=None
 
 
 def set_project(project_qualified_name):
-    """Setups `Neptune client library <https://github.com/neptune-ml/neptune-client>`_ to work with specific project.
+    """Setups `Neptune client library <https://github.com/neptune-ai/neptune-client>`_ to work with specific project.
```
Comment (hubertjaworskineptune) [inline neptune/__init__.py]: revert until github organization changes

File: neptune/exceptions.py [hunk 4]
```diff
@@ -24,7 +24,7 @@ class Uninitialized(NeptuneException):
     def __init__(self):
         super(Uninitialized, self).__init__(
             "You must initialize neptune-client first. "
-            "For more information, please visit: https://github.com/neptune-ml/neptune-client#initialize-neptune")
+            "For more information, please visit: https://github.com/neptune-ai/neptune-client#initialize-neptune")
```
Comment (hubertjaworskineptune) [inline neptune/exceptions.py]: revert until github organization changes

File: cli/__init__.py [hunk 5]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+# Copyright (c) 2020, Neptune Labs Sp. z o.o.
```
Comment (hubertjaworskineptune) [inline cli/__init__.py]: I don't like this change, I believe that the year should change when file is edited in next year, not globally (I might be wrong)

File: pylintrc [hunk 6]
```diff
@@ -22,7 +22,7 @@ persistent=yes
...
+file-header=#\n# Copyright \(c\) 2020, Neptune Labs Sp\. z o\.o\.\n#\n# Licensed under the Apache License, Version 2\.0 \(the "License"\);\n# you may not use this file except in compliance with the License\.\n# You may obtain a copy of the License at\n#\n#     http://www\.apache\.org/licenses/LICENSE-2\.0\n#\n# Unless required by applicable law or agreed to in writing, software\n# distributed under the License is distributed on an "AS IS" BASIS,\n# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied\.\n# See the License for the specific language governing permissions and\n# limitations under the License\.\n#\n
```
Comment (kamil-kaczmarek) [inline pylintrc]:
```suggestion
file-header=#\n# Copyright \(c\) (2019|2020), Neptune Labs Sp\. z o\.o\.\n#\n# Licensed under the Apache License, Version 2\.0 \(the "License"\);\n# you may not use this file except in compliance with the License\.\n# You may obtain a copy of the License at\n#\n#     http://www\.apache\.org/licenses/LICENSE-2\.0\n#\n# Unless required by applicable law or agreed to in writing, software\n# distributed under the License is distributed on an "AS IS" BASIS,\n# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied\.\n# See the License for the specific language governing permissions and\n# limitations under the License\.\n#\n
```

File: docs/conf.py [hunk 7]
```diff
@@ -18,8 +18,8 @@
 # -- Project information -----------------------------------------------------
 
 project = 'Neptune client'
-copyright = '2019, team neptune.ml'
```
Comment (hubertjaworskineptune) [inline docs/conf.py]: I believe that this year should not be modified as well

File: docs/conf.py [hunk 8]
```diff
@@ -18,8 +18,8 @@
...
-author = 'team neptune.ml'
+copyright = '2020, team neptune.ai'
```
Comment (kamil-kaczmarek) [inline docs/conf.py]:
```suggestion
copyright = '2019, team neptune.ai'
```

File: docs/conf.py (hunk 7 above)
Comment (kamil-kaczmarek) [reply docs/conf.py]: commited changes

File: README.md (hunk 1 above)
Comment (kamil-kaczmarek) [reply README.md]: done

File: neptune/__init__.py (hunk 2 above)
Comment (kamil-kaczmarek) [reply neptune/__init__.py]: done

**Pull request merged**